kclvm/sema/src/core/package.rs

```diff
@@ -40,22 +40,28 @@ impl PackageDB {
         self.module_info.get(name)
     }
 }
-#[derive(Debug)]
+#[derive(Debug, Clone)]
 pub struct PackageInfo {
     pub(crate) fully_qualified_name: String,
     pub(crate) filename: String,
+    pub(crate) kfile_paths: IndexSet<String>,
```

Comment:
What does the `kfile_paths` and `filename` means?

Reply:
@NeverRaR

Reply:
kfile_paths is the path fo all kcl files under this package

filename represents the folder of the package itself

Reply:
> kfile_paths is the path fo all kcl files under this package
> 
> filename represents the folder of the package itself

Make sense. s/filename/pkg_filepath or add some documents?